### ISSUE TITLE
multiplication of ElasticArrays is broken

### ISSUE THREAD

**Issue by garrison** [JuliaArrays/ElasticArrays.jl#5]
Multiplying an ElasticArray matrix times a column vector (e.g. `ElasticArray{Float64}(3, 3) * [1,2,3]`) results in the following:

```
ERROR: conversion to pointer not defined for ElasticArrays.ElasticArray{Float64,2,1}
Stacktrace:
 [1] gemv!(::Char, ::Float64, ::ElasticArrays.ElasticArray{Float64,2,1}, ::Array{Float64,1}, ::Float64, ::Array{Float64,1}) at ./linalg/blas.jl:503
 [2] gemv!(::Array{Float64,1}, ::Char, ::ElasticArrays.ElasticArray{Float64,2,1}, ::Array{Float64,1}) at ./linalg/matmul.jl:266
 [3] *(::ElasticArrays.ElasticArray{Float64,2,1}, ::Array{Int64,1}) at ./linalg/matmul.jl:75
```

I believe this is a consequence of deriving from `DenseArray` while using the fallback `unsafe_convert`.

Combined with #4, this is causing my code to fail.

**Comment by oschulz** [comment]
Thanks, good catch.

**Comment by garrison** [comment]
Fixed in #6.
